routes/exportCSV.js

```diff
@@ -6,6 +6,7 @@ router.post('/boxes', async (req, res) => {
   try {
     return res.status(200).send(await exportCSVService.getBoxesByFilters(req.body));
   } catch (err) {
+    console.log(err);
```

Comment:
⚠️ **[eslint]** <[no-console](https://eslint.org/docs/rules/no-console)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected console statement.

Reply:
nit: can we remove this